en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -200,15 +204,72 @@ For instance, the ``Redirection`` entity will look like:
         // ...
     }
 
-``type`` will take ``string`` as a default parameter.
-``name`` will take the name of the property if not set.
+Yaml:
+
+.. code-block:: yaml
+
+    # src/AppBundle/Resources/config/search/Redirection.yml
+
+    AppBundle\Document\AppBundle\Document:
+        properties:
+            locale:
+                key: locale
+            siteName:
+                key: site_name
+            routePattern:
+                key: route_pattern
+            url:
+                key: [redirection, url]
+            permanent:
+                key: permanent
+                type: boolean
+
+Xml:
+
+.. code-block:: xml
+
+    <!-- src/AppBundle/Resources/config/search/Redirection.xml -->
+
+    <search-mapping>
+        <class name="AppBundle\Document\AppBundle\Document">
+            <field field="locale" key="locale" />
+            <field field="siteName" key="site_name" />
+            <field field="routePattern" key="route_pattern" />
+            <field field="url" key="redirection, url" />
+            <field field="locale" key="locale" />
+            <field field="permanent" key="permanent" type="boolean"/>
+            <field field="permanent" key="fake_property2, fake_property_multi"/>
```

Comment:
fake_property is a naming for test, in a doc you should use something more explicit like a concrete example